qiskit/providers/ibmq/experiment/experiment.py

```diff
@@ -43,7 +44,9 @@ def __init__(
             analysis_results: Optional[List[AnalysisResult]] = None,
             hub: Optional[str] = None,
             group: Optional[str] = None,
-            project: Optional[str] = None
+            project: Optional[str] = None,
+            share_level: Union[ExperimentShareLevel, str] = None,
```

Comment:
```suggestion
            share_level: Optional[Union[ExperimentShareLevel, str]] = None,
```
I'm surprised mypy didn't complain - if it can be `None` the type needs to have `Optional`.